tutor4points/core/templates/home.html

```diff
@@ -44,14 +44,21 @@ <h2>Tutor Requests</h2>
                                         {{tutor_request.tutor_date}} @<br>{{tutor_request.tutor_time}} <br/>
                                     </div>
                                     <div class = "respond-btns d-flex">
-                                        <button type="submit" name="accept-request" class="btn-secondary respond-button" 
-                                            data-toggle= "modal" data-target="#modal-accept-request">
-                                            Accept
-                                        </button>
-                                        <button type="submit" name="decline-request" class="btn-secondary respond-button" 
-                                        data-toggle= "modal" data-target="#modal-decline-request">
-                                            Decline
-                                        </button>
+                                        {% if tutor_request.accepted %}
+                                            <button type="submit" name="btn-accept-request" cless="btn-secondary"
+                                                data-toggle="modal" data-target="#modal-complete-request">
```

Comment:
It'll be helpful to still have the status next to the button or above it, so the user knows the request has been accepted.